cmake/OpenCVDetectInferenceEngine.cmake

```diff
@@ -62,9 +62,7 @@ function(add_custom_ie_build _inc _lib _lib_rel _lib_dbg _msg)
     if(find_prefix STREQUAL "_empty_")  # foreach doesn't iterate over empty elements
       set(find_prefix "")
     endif()
-    foreach(find_suffix ${CMAKE_FIND_LIBRARY_SUFFIXES})
-      ocv_ie_find_extra_libraries("${find_prefix}" "${find_suffix}")
-    endforeach()
+    ocv_ie_find_extra_libraries("${find_prefix}" ".${CMAKE_SHARED_LIBRARY_SUFFIX}")
```

Comment:
This variable already has dot prefix:
```
Linux: CMAKE_SHARED_LIBRARY_SUFFIX=.so
Mac: CMAKE_SHARED_LIBRARY_SUFFIX=.dylib
Windows: CMAKE_SHARED_LIBRARY_SUFFIX=.dll
```

On Windows it just wrong (must be `.lib`).

Reply:
fixed.